js/bybit.js

```diff
@@ -4012,15 +4011,68 @@ module.exports = class bybit extends Exchange {
             currency = this.currency (code);
             request['coin'] = currency['id'];
         }
+        const depositMethodCutoff = 1626393600000;
+        let method = 'privateGetAssetV1PrivateDepositRecordQuery';
+        const until = this.safeNumber (params, 'until');
         if (since !== undefined) {
-            request['start_date'] = this.yyyymmdd (since);
+            request['wallet_fund_type'] = 'Deposit'; // Deposit, Withdraw, RealisedPNL, Commission, Refund, Prize, ExchangeOrderWithdraw, ExchangeOrderDeposit
+            if (since <= depositMethodCutoff) {
+                method = 'privateGetV2PrivateWalletFundRecords';
+                request['start_date'] = this.yyyymmdd (since);
+            } else {
+                const startTime = since / 1000;
+                request['start_time'] = startTime;
+                const thirtyDays = 2592000;
+                if (until === undefined) {
+                    request['end_time'] = this.sum (startTime, thirtyDays);
+                } else if (((startTime / 1000) - startTime) > thirtyDays) {
+                    throw new BadRequest (this.id + ' fetchDeposits () time between since and params["until"] cannot be longer than 30 days');
+                }
+            }
+        }
+        if (until !== undefined) {
+            if (until <= depositMethodCutoff) {
+                method = 'privateGetV2PrivateWalletFundRecords';
+                request['end_date'] = this.yyyymmdd (until);
+            } else {
+                request['end_time'] = until / 1000;
```

Comment:
Do they accept fractional numbers here?

Reply:
fixed in https://github.com/ccxt/ccxt/pull/15116/commits/493ea49e91b8dcbc3ea83784a9aaf7c67622e56e